kolibri/logger/models.py

```diff
@@ -30,45 +33,56 @@ class BaseLogModel(models.Model):
         )
     )
 
-    user = models.ForeignKey(FacilityUser, blank=True, null=True, db_index=True)
-
     class Meta:
         abstract = True
 
+    def infer_dataset(self):
+        if self.user:
+            return self.user.dataset
+        else:
+            facility = Facility.get_default_facility()
+            assert facility, "Before you can save logs, you must have a facility"
+            return facility.dataset
 
-class ContentInteractionLog(BaseLogModel):
+
+class ContentSessionLog(BaseLogModel):
```

Comment:
I think this name change is misleading.

An interaction can be defined by the particular content renderer - so in the case of a video it could well be the individual viewing session. However, in the case of an exercise, there will almost certainly be multiple interactions per sessions.

I would prefer to retain the old nomenclature, even if, in some circumstances, for some renderers, an interaction *is* a session.